lib/index.d.ts

```diff
@@ -0,0 +1,286 @@
+// Type definitions for stardog.js 0.3.1
+// Project: https://github.com/stardog-union/stardog.js
+// Definitions by: Stephen Nowell <http://github.com/snowell>, Ty Soehngen <http://github.com/tysoeh>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+import Headers from 'fetch-ponyfill';
+import Guid from '@types/node';
+
+declare global {
+  
+  /** stardog.js: The Stardog JS API*/
+  namespace Stardog {
+
+    type ContentType =
+        'application/x-turtle' |
+        'text/turtle' |
+        'application/rdf+xml' |
+        'text/plain' |
+        'application/x-trig' |
+        'text/x-nquads' |
+        'application/trix';
+    
+    type MIMEType =
+        'text/plain' |
+        'application/json' |
+        'text/boolean';
+
+    interface HTTPOptions {
+        headers?: Headers;
```

Comment:
That being said, what is this for? Looks like you're using it in transaction stuff. The only exposed options there are "content-type" and "encoding". Don't think you need a whole interface for it.